### PR TITLE
[JW8-11947] Add PiP on the right click menu

### PR DESCRIPTION
**Pull request by pajong**
### This PR will...
- Add PiP button on the right click menu that toggles PiP mode

#### Addresses Issue(s):
JW8-11947

### Checklist
- [ ] Jenkins builds and unit tests are passing
- [ ] I have reviewed the automated results

### CODE REVIEW

**Comment by jwrobot-player** [comment]
[:warning:](https://jenkins-player.longtailvideo.com/job/jw7-pr-multi-opensource/1946/) MULTI Build for commit 3bdb8f73f5ce40a93af5e04a5c4a6195ea91efe9 is unstable (UNSTABLE).
[:building_construction:](https://jenkins-player.longtailvideo.com/job/jw7-opensource/9379/)[:black_small_square:](https://jenkins-player.longtailvideo.com/job/jw7-opensource/9379/consoleFull) jwplayer build SUCCESS
[:clipboard:](https://jenkins-player.longtailvideo.com/job/jw7-opensource-unit-tests/6313/)[:black_small_square:](https://jenkins-player.longtailvideo.com/job/jw7-opensource-unit-tests/6313/consoleFull) jwplayer unit tests SUCCESS
[:building_construction:](https://jenkins-player.longtailvideo.com/job/jw7-commercial/28507/)[:black_small_square:](https://jenkins-player.longtailvideo.com/job/jw7-commercial/28507/consoleFull) jwplayer-commercial build SUCCESS
[:clipboard:](https://jenkins-player.longtailvideo.com/job/jw7-commercial/23942/)[:black_small_square:](https://jenkins-player.longtailvideo.com/job/jw7-commercial-unit-tests/23942/consoleFull) jwplayer-commercial unit tests SUCCESS
[:cucumber: Cucumber Integration Tests](https://jenkins-player.longtailvideo.com/job/jw7-commercial-cucumber-wdio/6103/allure/) UNSTABLE
[:jack_o_lantern:](http://player-pr-test-jenkins.longtailvideo.com/builds/28507/archive/test/squash/) Manual Tests
[:tv:](http://player-pr-test-jenkins.longtailvideo.com/builds/28507/archive/test/view/) Views

File: src/js/view/controls/rightclick.js [hunk 1]
```diff
@@ -62,6 +62,12 @@ export default class RightClick {
                 type: 'keyboardShortcuts'
             });
         }
+        if (!OS.mobile && model.get('pipIcon') !== 'disabled' && (Browser.chrome || Browser.edge || Browser.safari)) {
```
Comment (mamaddox) [inline src/js/view/controls/rightclick.js:65]: Could set
`this.pipMenu = !OS.mobile && model.get('pipIcon') !== 'disabled' && (Browser.chrome || Browser.edge || Browser.safari)`
if we wanted, but LGTM!

**Review by mamaddox — APPROVED** [review]

**Review by zetagame — APPROVED** [review]

File: src/js/view/controls/rightclick.js (hunk 1 above)
Comment (pajong) [reply src/js/view/controls/rightclick.js:65]: Thank you! I will make this change before merging

**Comment by jwrobot-player** [comment]
[:warning:](https://jenkins-player.longtailvideo.com/job/jw7-pr-multi-opensource/1951/) MULTI Build for commit e77a0fe181bf975d5c5e5bd776e4e278ca105f60 is unstable (UNSTABLE).
[:building_construction:](https://jenkins-player.longtailvideo.com/job/jw7-opensource/9386/)[:black_small_square:](https://jenkins-player.longtailvideo.com/job/jw7-opensource/9386/consoleFull) jwplayer build SUCCESS
[:clipboard:](https://jenkins-player.longtailvideo.com/job/jw7-opensource-unit-tests/6318/)[:black_small_square:](https://jenkins-player.longtailvideo.com/job/jw7-opensource-unit-tests/6318/consoleFull) jwplayer unit tests SUCCESS
[:building_construction:](https://jenkins-player.longtailvideo.com/job/jw7-commercial/28518/)[:black_small_square:](https://jenkins-player.longtailvideo.com/job/jw7-commercial/28518/consoleFull) jwplayer-commercial build SUCCESS
[:clipboard:](https://jenkins-player.longtailvideo.com/job/jw7-commercial/23951/)[:black_small_square:](https://jenkins-player.longtailvideo.com/job/jw7-commercial-unit-tests/23951/consoleFull) jwplayer-commercial unit tests SUCCESS
[:cucumber: Cucumber Integration Tests](https://jenkins-player.longtailvideo.com/job/jw7-commercial-cucumber-wdio/6112/allure/) UNSTABLE
[:jack_o_lantern:](http://player-pr-test-jenkins.longtailvideo.com/builds/28518/archive/test/squash/) Manual Tests
[:tv:](http://player-pr-test-jenkins.longtailvideo.com/builds/28518/archive/test/view/) Views

**Pull request merged**